docs/ops/doc/examples/example_flim_analysis.rst

```diff
@@ -36,17 +36,15 @@ Script execution requires a number of parameters, which may be useful for adapti
 | Bin Kernel Radius                    | 1     |
```

Comment:
Any reason not to bake all of these as default values in the script? You're already doing it for lifetime axis.. I don't think it takes away from reusability on other data and will make it overall more foolproof for people who want to reproduce this example.